cc7-reana/README.md

```diff
@@ -0,0 +1,25 @@
+ALICE container for REANA
+=========================
+
+Purpose of this container is to make it possible to reproduce a certain ALICE
+analysis with no external requirements. The container will be created with a
+certain AliPhysics version and all of its dependencies automatically. To build
+it:
+
+    docker build . --build-arg ALIPHYSICS_VERSION=vAN-20170101-1 -t aliphysics-vAN-20170101-1
```

Comment:
You may want to use `-t aliphysics-van-20170101-1` i.e. with `s/vAN/van/g` to have repository name all lowercase here and in other CLI examples below -- otherwise one would get "invalid reference format: repository name must be lowercase".